Resources/Prototypes/_NF/Catalog/Fills/Items/briefcases.yml

```
@@ -75,7 +75,7 @@
 - type: entity
   parent: SpaceCashExpedition
   id: SpaceCashExtremeExpedition
-  name: spesos (70000)
+  name: spesos (70.000)
```

Consistency with en-US cultureinfo.
```suggestion
  name: spesos (70,000)
```